ZividOpenCV/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -25,32 +27,64 @@ int main()
         std::cout << "Connecting to the camera" << std::endl;
         auto camera = zivid.connectCamera();
 
-        std::cout << "Configuring the camera settings" << std::endl;
-        camera << Zivid::Settings::Iris{ 21 } << Zivid::Settings::ExposureTime{ std::chrono::microseconds{ 20000 } }
-               << Zivid::Settings::Gain{ 1 } << Zivid::Settings::Brightness{ 1.0 };
+        std::cout << "Enter <a><Return> for 2D only, and just <Return> for 3D" << std::endl;
+        const auto command = getInput();
+        bool use2D = false;
+        if(command == "a" || command == "A")
+        {
+            use2D = true;
+        }
+
+        cv::Mat rgb;
+
+        if(use2D)
+        {
+            std::cout << "2D only" << std::endl;
+
+            std::cout << "Configuring the camera settings" << std::endl;
+            auto settings = Zivid::Settings2D();
+            settings.set(Zivid::Settings2D::ExposureTime{ std::chrono::microseconds{ 10000 } });
+            settings.set(Zivid::Settings2D::Gain{ 1.0 });
+            settings.set(Zivid::Settings2D::Iris{ 35 });
+
+            std::cout << "Capture a 2D frame" << std::endl;
+            const auto frame = camera.capture2D(settings);
 
-        std::cout << "Capturing a frame" << std::endl;
-        const auto frame = camera.capture();
+            std::cout << "Get RGBA8 image from Frame2D" << std::endl;
```

Comment:
You should be consistent with verb usage, the rest of the sample is using the continuous aspect ("Getting"  instead of "Get"). Same with "Capturing" vs "Capture" above.